kite/tests.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+import numpy as np
+from kite.gates import Gates
+
+
+class test_gates(unittest.TestCase):
+    
+    def test_base_gate_matrices(self):
+        "Test the gates which are hardcoded matrices"
+        g = Gates()
+        assert np.isclose(g.I, np.eye(2)).all()
```

Comment:
I'm not sure if  we need to test hardcoded matrices. If this test was already there, we need to delete it.

Reply:
I see it now, I put "pass" instead, but left the test module there for the future as a best practice.